src/nni_manager/package.json

```diff
@@ -26,7 +26,8 @@
     "ts-deferred": "^1.0.4",
     "typescript-ioc": "^1.2.4",
     "typescript-string-operations": "^1.3.1",
-    "webhdfs": "^1.2.0"
+    "webhdfs": "^1.2.0",
+    "deepmerge": "^4.2.2"
```

Comment:
Looks like your are not using `yarn add`

Reply:
> Looks like your are not using `yarn add`

fixed.